src/components/Article.js

```diff
@@ -156,8 +159,14 @@ const ArticleInner = ({ lang, articleTitle, initialAnchor }) => {
     showQuickFactsPopup({ article })
   }
 
-  const showGallery = () => {
-    showGalleryPopup({ items: article.media })
+  const showGallery = fileName => {
+    let startIndex = 0
+
+    if (fileName) {
+      startIndex = article.media.findIndex(media => media.canonicalizedTitle === fileName)
+    }
+
+    showGalleryPopup({ items: article.media, startIndex })
```

Comment:
You could give the component the target filename and let it figure out the index so you would have to duplicate the findIndex here and in quick facts.